Exception trying to log out from the UI

You still get logged out internally, the UI just doesn't update.

```
00:00:15.275 [ERROR] Encountered an exception published to an object with an unobserved ThrownExceptions property|System.InvalidOperationException: Sequence contains no elements
   at System.Linq.ThrowHelper.ThrowNoElementsException()
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at NexusMods.Networking.NexusWebApi.LoginManager.Logout() in NexusMods.App/src/Networking/NexusMods.Networking.NexusWebApi/LoginManager.cs:line 150
   at NexusMods.App.UI.Controls.TopBar.TopBarViewModel.Logout() in NexusMods.App/src/NexusMods.App.UI/Controls/TopBar/TopBarViewModel.cs:line 123
```

--- a/src/Networking/NexusMods.Networking.NexusWebApi/LoginManager.cs
+++ b/src/Networking/NexusMods.Networking.NexusWebApi/LoginManager.cs
@@ -147,8 +147,8 @@ public sealed class LoginManager : IDisposable, ILoginManager
     /// </summary>
     public async Task Logout()
     {
-        await _jwtTokenRepository.Delete(_jwtTokenRepository.All.First());
         _cachedUserInfo.Evict();
+        await _jwtTokenRepository.Delete(_jwtTokenRepository.All.First());
     }
 
     /// <inheritdoc/>

